include/drivers/pwm.h

```diff
@@ -22,13 +23,45 @@
 #include <errno.h>
 #include <zephyr/types.h>
 #include <stddef.h>
+#include <sys/math_extras.h>
 #include <device.h>
 #include <dt-bindings/pwm/pwm.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/**
+ * @name PWM capture configuration flags
+ * @{
+ */
+
+/** PWM pin capture captures period. */
+#define PWM_CAPTURE_TYPE_PERIOD		(1 << 1)
```

Comment:
Is bit 0 reserved for some reason?

Reply:
Yes, it is used for the polarity (`PWM_POLARITY_NORMAL`/`PWM_POLARITY_INVERTED`), which can be set from DTS.

Reply:
Ah.  OK, a comment to that effect could help (or defining `PWM_CAPTURE_POLARITY_foo` as aliases, since the devicetree constant has a different prefix).